api/package.json

```diff
@@ -28,7 +28,8 @@
     "lint-fix": "eslint --fix --ext ts \"./src/**/*.ts\"",
     "lint:lockfile": "lockfile-lint --path yarn.lock --type yarn --validate-https --allowed-hosts npm yarn",
     "format": "prettier --write --end-of-line=auto \"./**/*.{js,jsx,ts,tsx,html,css,json}\" --ignore-path .gitignore",
-    "format:check": "prettier --check --end-of-line=auto \"./**/*.{js,jsx,ts,tsx,html,css,json}\" --ignore-path .gitignore"
+    "format:check": "prettier --check --end-of-line=auto \"./**/*.{js,jsx,ts,tsx,html,css,json}\" --ignore-path .gitignore",
+    "docs": "typedoc --plugin typedoc-plugin-markdown --options typedoc.json"
```

Comment:
is this run by the CI? and is `typedoc.json` provided by the action too? (just asking since i tried running it locally and it didn't work :P )

Reply:
Whoopsie, didn't see you commented here. I should have removed it since the generation is managed by typedocusaurus, not here anymore.